mautic/mautic
# Added an option for excluding segments when creating/updating segment emails

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [❌]
| New feature/enhancement? (use the a.x branch)      | [✅]
| Deprecations?                          | [❌]
| BC breaks? (use the c.x branch)        | [❌]
| Automated tests included?              | [✅] 
| Related user documentation PR URL      |  
| Related developer documentation PR URL |  
| Issue(s) addressed                     |  



#### Description:

As a user of Mautic, I want to exclude certain email addresses for segment emails, so that I can save time by avoiding all the steps required to go through a Campaign to exclude certain emails. This will prevent the need for extra segments built to handle exclusion cases

#### Steps to test this PR:


1. Create at least two segments with a couple of contacts.
2. Create a new segment email and select one segment in _Contact segment_ field and another segment in _Exclude contact segment_ field.
3. Save and send the email.
4. The contacts belonging to the excluded segment should not receive any email.
